deploy/olm-catalog/tektoncd-pipeline-operator/0.0.1/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -3,6 +3,12 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     capabilities: Basic Install
+    categories: "Developer Tools"
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/rhpipeline/nikthoma_tektoncd-pipeline-operator:v0.2.0-2
```

Comment:
we should change to the new org :(

Reply:
updated to quay.io/openshift-pipeline  org